src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(cascade = {CascadeType.MERGE})
+    private Movie movie;
+    @ManyToOne(cascade = {CascadeType.MERGE})
+    private CinemaHall cinemaHall;
```

Comment:
There is no need to use cascades if this is not in the problem statement. Also add @JoinColumn for cinemaHall and @Column for showTime

Reply:
Ok Understood